packages/now-cli/test/integration.js

```diff
@@ -519,6 +519,26 @@ test('Deploy `api-env` fixture and test `now env` command', async t => {
   await nowEnvLsIsEmpty();
 });
 
+test('deploy with metadata with = as the value', async t => {
+  const target = fixture('redirects-v2');
+
+  const { exitCode, stderr, stdout } = await execa(
+    binaryPath,
+    [target, ...defaultArgs, '--confirm', '--meta', 'someKey=='],
```

Comment:
Frankly, I'm not sure this is the best way to write a test case for this. 
Guide me if this is not the correct way.